stdlib/template/distutils.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+"""Provides an exhaustive template that downloads, builds and wraps a software based on ``python``'s ``distutils``.
+"""
+
+import os
+import stdlib
+import stdlib.fetch
+import stdlib.extract
+import stdlib.patch
+import stdlib.split.drain_all
+import stdlib.deplinker.elf
+from multiprocessing import cpu_count
+
+
+def distutils_build(
+    *args: str,
+    python_binary: str = 'python3',
+    script_path: str = './setup.py',
+    fail_ok: bool = True,
+):
+    """Run ``python3 ./setup.py build``.
+
+    :param args: Any extra arguments to give to ``setup.py``
+    :param python_binary: The command or path to use. The default value is ``python3``.
+    :param script_path: The path to the distutils script. The default value is ``./setup.py``.
+    :param fail_ok: If ``True``, the execution is aborted if ``cargo`` fails.
```

Comment:
Cargo, really ?
Also, you document the default value as being `False`, but you set it to `True`

Reply:
Woops